src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,34 @@
 import React from 'react';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+import { Comment } from '../../types/Comment';
```

Comment:
You should add files with TS types as well

Reply:
fixed